py/stackctrl.h

```diff
@@ -26,15 +26,20 @@
 #ifndef MICROPY_INCLUDED_PY_STACKCTRL_H
 #define MICROPY_INCLUDED_PY_STACKCTRL_H
 
-#include "py/mpconfig.h"
+#include "py/mpstate.h"
```

Comment:
I was a bit worried this would introduce a circular header dependency, but luckily it does not.

So, now it would also be possible to make `mp_stack_set_top()` static-inline, which should also help to reduce code size.

Reply:
Have pushed a commit here to do this as well.